### ISSUE TITLE
Nexus server container fails to boot due to java OOM 

### ISSUE THREAD

**Issue by jordigilh** [redhat-cop/rhel-edge-automation-arch#184]
While running the bootstrap PR from @sabre1041(#181) , I noticed that the container that holds the nexus server was restarting and never reaching stability. Looking at the logs I discovered the JVM was complaining of OOM errors when loading jars
```
2021-12-04 23:35:25,271+0000 ERROR [FelixStartLevel] *SYSTEM Felix - Bundle org.sonatype.nexus.plugins.nexus-audit-plugin [232] Error starting reference:file:system/org/sonatype/nexus/plugins/nexus-audit-plugin/3.37.0-01/nexus-audit-plugin-3.37.0-01.jar (java.lang.OutOfMemoryError: Java heap space)
java.lang.OutOfMemoryError: Java heap space
	at org.apache.felix.resolver.util.OpenHashMap.clone(OpenHashMap.java:820)
	at org.apache.felix.resolver.util.OpenHashMapList.deepClone(OpenHashMapList.java:35)
	at org.apache.felix.resolver.Candidates.copy(Candidates.java:1143)
	at org.apache.felix.resolver.Candidates.permutate(Candidates.java:1196)
	at org.apache.felix.resolver.ResolverImpl$ResolveSession.permutateIfNeeded(ResolverImpl.java:155)
	at org.apache.felix.resolver.Candidates.checkSubstitutes(Candidates.java:344)
	at org.apache.felix.resolver.ResolverImpl.checkConsistency(ResolverImpl.java:606)
	at org.apache.felix.resolver.ResolverImpl.findValidCandidates(ResolverImpl.java:574)
	at org.apache.felix.resolver.ResolverImpl.doResolve(ResolverImpl.java:437)
	at org.apache.felix.resolver.ResolverImpl.resolve(ResolverImpl.java:420)
	at org.apache.felix.resolver.ResolverImpl.resolve(ResolverImpl.java:374)
	at org.apache.felix.framework.StatefulResolver.resolve(StatefulResolver.java:478)
	at org.apache.felix.framework.Felix.resolveBundleRevision(Felix.java:4363)
	at org.apache.felix.framework.Felix.startBundle(Felix.java:2281)
	at org.apache.felix.framework.Felix.setActiveStartLevel(Felix.java:1539)
	at org.apache.felix.framework.FrameworkStartLevelImpl.run(FrameworkStartLevelImpl.java:308)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by sabre1041** [comment]
I noticed this issue as well

**Comment by jordigilh** [comment]
Fixed in catalog source 2.0.6. 
To test this delete the operator image from the worker nodes and redeploy the operator.
